src/hmem.c

```diff
@@ -1,4 +1,6 @@
 /*
+ * Copyright (c) 2004, 2005 Topspin Communications.  All rights reserved.
+ * Copyright (c) 2006 Cisco Systems, Inc.  All rights reserved.
```

Comment:
Since this code comes from https://github.com/linux-rdma/rdma-core/blob/master/libibverbs/memory.c, I think the copyright needs to be updated. Thoughts?

Reply:
Can you locate the patch that added this there and identify the author?

Reply:
I think this is it: https://github.com/linux-rdma/rdma-core/commit/422642f1cf021b9de1f445f4392c99d3c2fe6624

Reply:
I think yes.  The author of the changes listed ibm as an email.  Topspin didn't exist by the time they were added.